src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -5572,15 +5572,15 @@ Map {
             "selectAllAria": Object {
               "type": "string",
             },
-            "selectDateRangeLabel": Object {
-              "type": "string",
-            },
```

Comment:
To avoid a breaking change we need to leave this in and deprecate it with a warning and map it to the new `selectTimeRangeLabel ` prop.

Reply:
on this one we agreed that since this prop only existed for a build or two, we were okay with the rename